.github/workflows/ubuntu-windows.yml

```diff
@@ -14,59 +14,11 @@ env:
   GITHUB_TOKEN: ${{ github.token }}
 
 jobs:
-
-  build_sirius:
-    name: ${{ matrix.os }} build sirius
-    runs-on: ${{ matrix.os }}
-    env:
-      Sirius_Ref: antares_integration
-    strategy:
-      matrix:
-        os: [windows-2019, ubuntu-20.04]
-    steps:
-      - name: Get release
-        if: github.event_name == 'release' && github.event.action == 'created'
-        id: get_release
-        uses:
-          bruceadams/get-release@v1.2.3
-
-      - name: install zip for windows
-        if: ${{ startsWith(matrix.os, 'windows') }}
-        shell: cmd
-        run: |
-          choco install zip --no-progress
-
-      - name: Checkout or-tools
-        uses: actions/checkout@v2
-
-      - name: setup sirius
-        id: setup
-        uses: ./.github/workflows/build-sirius
-        with:
-          sirius-ref: ${{ env.Sirius_Ref }}
-
-      - name: Upload sirius install
-        uses: actions/upload-artifact@v2
-        with:
-          name: ${{ matrix.os }}_sirius
-          path: ${{ steps.setup.outputs.sirius-install-path }}
-
-      - name: Publish sirius install
-        if:  ${{ github.event_name == 'release' && github.event.action == 'created' }}
-        uses: actions/upload-release-asset@v1.0.2
-        with:
-          upload_url: ${{ steps.get_release.outputs.upload_url }}
-          asset_path: ${{ steps.setup.outputs.sirius-archive-path }}
-          asset_name: ${{ matrix.os }}_${{ steps.setup.outputs.sirius-archive-name }}
-          asset_content_type: application/zip
-
-
   build:
-    needs: [build_sirius]
     name: ${{ matrix.os }} shrd=${{ matrix.shared }} sirius=${{ matrix.sirius }} xprs=${{ matrix.xprs }} extras=${{ matrix.extras }} java=${{ matrix.java }} dotnet=${{ matrix.dotnet }} python=${{ matrix.python }}-${{ matrix.python-version }}
     runs-on: ${{ matrix.os }}
     env:
-      Sirius_Ref: antares_integration
+      SIRIUS_RELEASE_TAG: antares-integration-v1.0
       XPRESSDIR: ${{ github.workspace }}/xpressmp813
       XPRESS: ${{ github.workspace }}/xpressmp813/bin
       XPRS_LIB_PATH: ${{ github.workspace }}/xpressmp813/lib
```

Comment:
```suggestion
      XPRESS: ${XPRESSDIR}/bin
      XPRS_LIB_PATH: ${XPRESSDIR}/lib
```

Reply:
it's not possible. The variables don't seem to be available at the level of line 23